draft-cutler-httpbis-partitioned-cookies.md

```diff
@@ -66,6 +66,9 @@ In order to increase privacy on the web, browser vendors are either planning or
 Although third-party cookies can enable third-party sites to track user behavior across different top-level sites, there are some cookie use cases on the web today where cross-domain subresources require some notion of session or persistent state that is scoped to a user's activity on a single top-level site.
 
 In order to meet these use cases, this document proposes changes to RFC6265bis to support a new cookie attribute, Partitioned, which restricts the contexts that a cookie is available to only those whose top-level document is same-site with the top-level document that the cookie was created in.
+
+When top-level site is the one setting the cookie, a calculation will be made to determine if the frame the cookie is being set on has any cross-site ancestors. If a cross-site ancestor is present, then that will be indicated in the key of the Partitioned attribute.
```

Comment:
Paraphrasing from the [storage partitioning repo](https://github.com/privacycg/storage-partitioning#additional-keying) a bit here:

"In addition to double-keying cookies on top-level site, we propose to additionally key cookies on a bit indicating whether the frame tree includes a cross-site ancestor to prevent attacks between framed documents."